services/web/client/source/class/qxapp/statics/Node.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * oSPARC - The SIMCORE frontend - https://osparc.io
+ * Copyright: 2019 IT'IS Foundation - https://itis.swiss
+ * License: MIT - https://opensource.org/licenses/MIT
+ * Authors: Ignacio Pascual (ignapas)
+ */
+
+qx.Class.define("qxapp.statics.Node", {
```

Comment:
Coud you, please, rename it to NodeIcon or something else?

Reply:
Done